### ISSUE TITLE
Incorrect quantized model ssd_mobilenet_v1_quantized_coco

### ISSUE THREAD

**Issue by cumberb1tch** [tensorflow/models#6112]
Object Detection model (ssd_mobilenet_v1_quantized_coco) from [this](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md#coco-trained-models) page is not fine-tuned with FakeQuantization.

Please see the Screen Shoot of model part: http://prntscr.com/me5nm6

We have pattern Conv ->  ReLU6 -> FakeQuantWithMinMaxVars -> Conv

And FakeQuantWithMinMaxVars have min and max range larger than 0-6 that comes from ReLU6. Which is impossible for trained model. Other quantized SSDs have ranges equal or smaller than 0-6 after ReLU6.

+ there are a lot of issues that this model shows low accuracy.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
